plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -377,17 +376,16 @@ open class HtmlRenderer(
     private fun FlowContent.buildRow(
         node: ContentGroup,
         pageContext: ContentPage,
-        sourceSetRestriction: Set<DisplaySourceSet>?,
-        style: Set<Style>
```

Comment:
The problem is that they don't do anything, it's misleading. If I'm using a function and I see that it takes a `styles` parameter, I expect that it will be used somewhere. If you pass styles here, it will not be applied, there's no logic for it. 

So what's the point of having an unused parameter that confuses developers and we're not even sure when/if it's going to be needed? It will take you 5 seconds to add it back :)

Reply:
It's also an anti-pattern as it turns out: https://en.wikipedia.org/wiki/Lava_flow_(programming)